src/resources/views/crud/buttons/create.blade.php

```diff
@@ -1,5 +1,5 @@
 @if ($crud->hasAccess('create'))
     <a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" data-style="zoom-in">
-        <span><i class="la la-plus"></i> {{ trans('backpack::crud.add') }} {{ $crud->entity_name }}</span>
+        <span><i class="la la-plus"></i> <span class="create-button-text-span">{{ trans('backpack::crud.add') }} {{ $crud->entity_name }}</span></span>
     </a>
```

Comment:
How about we add `bp-button="create"` to the anchor? That way you can style the button itself too, if needed.

Reply:
And it'd be similar to our other CSS hooks, `bp-section`

Reply:
https://backpackforlaravel.com/docs/6.x/crud-how-to#how-to-customize-crud-panel-design-css-hooks